src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1097,11 +1097,6 @@ static guint16 sri_vector_methods [] = {
 	SN_AsUInt16,
 	SN_AsUInt32,
 	SN_AsUInt64,
-	SN_AsVector128,
```

Comment:
These are dead code.

Reply:
Why is this removed ? These methods still exist in Vector128 etc. ?

Reply:
Currently, we are not emitting intrinsics for them at all. Could always add them back later when we emitting stuff for them.